src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -14,103 +14,150 @@ resources:
 
-```bash
+```shell
 docker run -it\
     -v $HOME/configs/:/mnt/deployer/configs/\
-    --entrypoint ruby deployer main.rb -c configs/[user].docker.local.json -d documentation/tutorial/user_stories/Hello/hello.json
+    --entrypoint ruby deployer main.rb -c configs/creds.json -d documentation/tutorial/user_stories/Hello/hello.json
 ```
 
-## hello.json
-The deployer is configuration driven, and [hello.json](https://github.com/newrelic/demo-deployer/blob/main/documentation/tutorial/user_stories/Hello/hello.json) is the default configuration for this example. Looking at this file, you will notice several interesting sections. 
-* services
-* global_tags
-* resources
-* instrumentation
-These section describe, to the deployer, the actions to take when it executes. For example, "global_tags" will, as the name implies, apply these tags to ALL the resources in your stack. This is how we apply the consistent tagging strategy discussed above. 
+With this command, you are running the `deployer` Docker container with the following options:
+
+- `-v` mounts the `$HOME/configs` volume (where you stored your configuration files) at `/mnt/deployer/configs/` in the container
+- `--entrypoint` sets the default command for Docker to `ruby`
+
+Finally, you pass a list of arguments to the entrypoint, including:
+
+- `-c configs/creds.json`: The user credentials configuration file
+- `-d documentation/tutorial/user_stories/Hello/hello.json`: The [deployment configuration file](https://github.com/newrelic/demo-deployer/tree/main/documentation/deploy_config)
+
+<Tip>
+
+You can specify the log level for the Deployer using `-l`. `info` is the default, but you can also use `debug` or `error`. This is a helpful option to use when something goes wrong.
+
+</Tip>
+
+Once you run this command, the Deployer will create and instrument some specific resources. This may take a few minutes. In the meantime, read on to learn more about what is actually happening during this process.
+
+#### Locating the user configuration files
+
+Notice, in the Docker command, that `-c` takes `configs/creds.json`. Also, you referenced your `.pem` file from `configs/`, as well.
+
+The reason for this is because you mounted `$HOME/configs` to `/mnt/deployer/configs/` and `/mnt/deployer/` is the working directory, according to the [Deployer's Dockerfile](https://github.com/newrelic/demo-deployer/blob/main/Dockerfile#L31). So, you can access your user configuration files at `configs/`, which is a relative path from your working directory.
+
+#### Understanding the deployment configuration file
+
+The deployment configuration file, [hello.json](https://github.com/newrelic/demo-deployer/blob/main/documentation/tutorial/user_stories/Hello/hello.json), which drives the Deployer, contains four sections:
+
+- `services`
+- `global_tags`
+- `resources`
+- `instrumentations`
+
+These sections describe, to the Deployer, the actions it should execute.
```

Comment:
To avoid choppiness: "These sections describe the actions the deployer executes." Also, should is mushy and not recommended, because it leaves open questions about when it will or will not execute the services.